src/wp-includes/comment-template.php

```diff
@@ -1740,6 +1740,8 @@ function comments_popup_link( $zero = false, $one = false, $more = false, $css_c
  *                              to addComment.moveForm(), and appended to the link URL as a hash value.
  *                              Default 'respond'.
  *     @type string $reply_text The text of the Reply link. Default 'Reply'.
+ *     @type string $reply_to_text The text template for "Reply to %s". Default 'Reply to %s'.
```

Comment:
```suggestion
 *     @type string $reply_text        The visible text of the Reply link. Default 'Reply'.
 *     @type string $reply_to_text     The accessible name of the Reply link, using `%s` as a placeholder
 *                                     for the comment author's name. Default 'Reply to %s'.
```

- Clarifies that `$reply_text` contains the _visible_ text for the link (and changing that from 'text' to 'HTML' might describe it better because HTML elements are accepted there).
- Uses the term 'accessible name' for `$reply_to_text`.
- Explains the comment author placeholder.
- Aligns parameter descriptions with `$use_reply_to_text`, though the other descriptions should also have more spaces to align with the longest parameter.